ensime_shared/ensime.py

```diff
@@ -49,7 +47,7 @@ class EnsimeClient(TypecheckHandler, DebuggerClient, ProtocolHandler):
     which stores the a handler per response type.
     """
 
-    def __init__(self, vim, launcher, config_path):
+    def __init__(self, vim, launcher, config_path):  # noqa: C901 FIXME
```

Comment:
Tools report that the cyclomatic complexity of `EnsimeClient.__init__` is too high—I completely agree, but refactoring will be a significant job for another day…